src/main/java/org/folio/dew/domain/dto/bursarfeesfines/AggregatedAccountsByUser.java

```diff
@@ -16,6 +16,11 @@ public class AggregatedAccountsByUser {
 
   public BigDecimal findTotalAmount() {
     BigDecimal totalAmount = new BigDecimal(0);
+
+    if (accounts == null) {
```

Comment:
Because in case accounts is null, the for loop afterwards would cause an error, and I want findTotalAmount() to work even when accounts is null. Should I keep that or do you think it is unnecessary?